### PR TITLE
Add Early Access Program to README.md

### PR DESCRIPTION
**Pull request by cemozerr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@
   <img alt="GitHub Workflow Status (with event)" src="https://img.shields.io/github/actions/workflow/status/sovereign-labs/sovereign-sdk/prerelease_checks.yml?label=Pre-release%20checks">
 </div>
 
+## NOTE: Early Access Program for Sovereign SDK
+
+To protect end-users against teams launching potentially insecure rollups at this alpha stage of Sovereign SDK, we’re continuing our development in a private repo. The SDK is always free and open-source, and once we get to testnet stage and iron out potential bugs, we will bring the updates to this main repo. If you want to build on the latest version, please fill out our Early Partner Program form to get access.
```
Comment (preston-evans98) [inline README.md]: Maybe we can clarify that "we will bring the updates to this main repo under the Apache 2.0 license"

**Review by preston-evans98 — APPROVED** [review]
Looks great overall. Suggested one minor clarification (that the license will still be Apache 2)

File: README.md (hunk 1 above)
Comment (cemozerr) [inline README.md]:
```suggestion
To protect end-users against teams launching potentially insecure rollups at this alpha stage of Sovereign SDK, we’re continuing our development in a private repo. The SDK is always free and open-source, and once we get to testnet stage and iron out potential bugs, we will bring the updates to this main repo under the Apache 2.0 license. If you want to build on the latest version, please fill out our Early Partner Program form to get access or reach out on Twitter!
```

Comment (cemozerr) [reply README.md]: Done.

**Pull request merged**